src/i965_drv_video.c

```diff
@@ -5935,6 +5935,12 @@ i965_QuerySurfaceAttributes(VADriverContextP ctx,
                   attribs[i].value.value.i = VA_FOURCC_I010;
                   i++;
                 }
+            } else if (obj_config->entrypoint == VAEntrypointEncPicture) {
+                attribs[i].type = VASurfaceAttribPixelFormat;
+                attribs[i].value.type = VAGenericValueTypeInteger;
+                attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE;
+                attribs[i].value.value.i = VA_FOURCC_YUY2;
```

Comment:
I think we're also missing Y8, UYVY, and RGB. I think adding all these will also necessitate increasing I965_MAX_SURFACE_ATTRIBUTES.